cmd/kube-proxy/app/server.go

```diff
@@ -217,7 +217,7 @@ func NewOptions() *Options {
 // Complete completes all the required options.
 func (o *Options) Complete() error {
 	if len(o.ConfigFile) == 0 && len(o.WriteConfigTo) == 0 {
-		klog.Warning("WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.")
+		klog.InfoS("WARNING: all flags other than --config, --write-config-to, and --cleanup are deprecated. Please begin using a config file ASAP.")
```

Comment:
Do we need period?

Reply:
Maybe period is ok in here. Because there are no para. Do you think so?

Reply:
I usually say no trailing periods on logs